turbo.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "pipeline": {
+    "build:clean": {
+      "inputs": ["lib/**/*", "dist/**/*"],
+      "outputs": []
+    },
+    "build:tsc": {
+      "dependsOn": ["^build:tsc"],
+      "inputs": ["src/**/*.ts", "tsconfig.json"],
+      "outputs": ["lib/**/*.d.ts"]
+    },
+    "build:types": {
+      "dependsOn": ["^build:types", "build:tsc"],
+      "inputs": ["lib/**/*.d.ts", "api-extractor.json"],
+      "outputs": ["etc/**/*", "dist/esm/index.d.ts"]
+    },
+    "build:esm": {
+      "dependsOn": [],
+      "inputs": ["src/**/*.ts"],
+      "outputs": ["dist/esm/**/*.js"]
+    },
+    "build:umd:cjs": {
+      "dependsOn": ["build:esm"],
+      "inputs": ["src/**/*.ts"],
+      "outputs": ["dist/cjs/**/*.js", "dist/umd/**/*.js"]
+    },
+    "test": {
+      "dependsOn": [],
+      "inputs": ["src/**/*.ts", "jest.config.ts"],
+      "outputs": []
+    },
+    "build": {
+      "dependsOn": ["build:esm", "build:umd:cjs", "build:types"],
+      "outputs": ["dist/**"]
+    },
+    "lint": {
+      "inputs": ["**/*.ts", "**/*.js"],
+      "outputs": []
+    },
+    "prepare": {
+      "dependsOn": ["^prepare"],
```

Comment:
I'm asking because I don't see any `prepare` command in packages, there's just one in the root `package.json`.

Reply:
Oh, sorry, it must have been in preparation for #670, it would require running a prepare script and order could matter. I can remove it and add it back if necessary.